src/LineageTree/lineageTree.py

```diff
@@ -2116,9 +2131,9 @@ def norm(*args):
         delta_tmp = partial(
             delta, corres1=corres1, corres2=corres2, times=self.cycle_time
         )
-        return uted(nodes1, adj1, nodes2, adj2, delta=delta_tmp) / norm(
-            nodes1, nodes2
-        )
+        return uted(
+            nodes1, adj1, nodes2, adj2, delta=delta_tmp
+        )  # / max(len(self.get_sub_tree(n1)),len(self.get_sub_tree(n2)))
```

Comment:
why removing my norm?
If you don't like it as it is, maybe you can provide a function that always return `1`?